### PR TITLE
move summary outside of a <table> in text/html

### PR DESCRIPTION
**Pull request by bkamins**
Fixes https://github.com/JuliaData/DataFrames.jl/issues/2800

### CODE REVIEW

File: src/abstractdataframe/io.jl [hunk 1]
```diff
@@ -188,6 +188,15 @@ function _show(io::IO, ::MIME"text/html", df::AbstractDataFrame;
...
 
     cnames = _names(df)[1:mxcol]
+    write(io, "<div class=\"data-frame\">")
+    if summary
+        omitmsg = if mxcol < size(df, 2)
+                      " (omitted printing of $(size(df, 2)-mxcol) columns)"
+                  else
+                      ""
+                  end
+        write(io, "<p>$(digitsep(nrow(df))) rows × $(digitsep(ncol(df))) columns$omitmsg</p>")
```
Comment (nalimilan) [inline src/abstractdataframe/io.jl:198]: Maybe simpler?
```suggestion
        write(io, "<p>$(digitsep(nrow(df))) rows × $(digitsep(ncol(df))) columns")
        if mxcol < size(df, 2)
            write(io, " (omitted printing of $(size(df, 2)-mxcol) columns)")
        end
        write(io, "</p>")
```

**Comment by nalimilan** [comment]
I trust you that this looks nice? :-)
(CI is failing)

**Comment by bkamins** [comment]
I did not have time to fix CI yet (to many things in parallel). I opened the PR to let #2800 verify if the tags/classes are as expected. I will finalize it soon.

File: src/abstractdataframe/io.jl (hunk 1 above)
Comment (bkamins) [reply src/abstractdataframe/io.jl]: done

**Comment by bkamins** [comment]
@nalimilan - this should be good to merge. I have re-written the tests and also tested under Jupyter Notebook.

@jkrumbiegel - checking if what we have here follows your expectations would be welcome. Thank you!

**Comment by jkrumbiegel** [comment]
Looking at the code, I think this is good. The outer div can also help when making the table scroll inside the div, in case it's too broad. This is weird to do in css if you only have the table.

**Review by nalimilan — APPROVED** [review]

**Comment by bkamins** [comment]
Thank you!

**Pull request merged**